packages/office-ui-fabric-react/src/components/Conformance.test.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import * as glob from 'glob';
+import * as path from 'path';
+
+/**
+ * These tests verify that Fabric components fulfill the following conditions:
+ *
+ *    1) The component accepts a className prop.
+ *    2) The component applies the className prop to some DOM element.
```

Comment:
by default the classname should exist on the root element. I think we would want to special case exclusions to that rule.

Reply:
By default, the tests look for the className on the root element, unless the component is added to the `classNameSelectors` object.